### PR TITLE
web/console: add marketplace modal layout

### PR DESCRIPTION
**Pull request by AnnaTsybel**

### CODE REVIEW

File: web/console/src/app/components/common/ModalMarketplace/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                                buy now
+                                <div className="marketplace-modal__buy-now__label">
+                                    for
+                                    <span>
+                                        {lot.currentPrice} {lot.currentPrice > ONE_COIN ? 'coins' : 'coin'}
+                                    </span>
+                                </div>
+                            </button>
+                           
+
```
Comment (TarasPotsikaylo) [inline web/console/src/app/components/common/ModalMarketplace/index.tsx]: delete

File: web/console/src/app/components/common/ModalMarketplace/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        const bidButton = () => { };
+
+        return <div className="marketplace-modal">
+            <div className="marketplace-modal__wrapper">
+                <div>
+                    <button onClick={() => setShowModal(false)} className="marketplace-modal__close-button">
+                        <img src={CloseModal} alt="modal close button" className="marketplace-modal__close-button__img" />
+                    </button>
+                </div>
+                <h2 className="marketplace-modal__title">{lot?.card.playerName}</h2>
```
Comment (TarasPotsikaylo) [inline web/console/src/app/components/common/ModalMarketplace/index.tsx]: what will be shown if there is no lot?
undefined?
Do we need to add some check?
Like this: 
_lot && lot.card.playerName ? lot?.card.playerName : 'not lot'_
_PS. Just double check this_

File: web/console/src/app/views/MarketPlacePage/index.tsx [hunk 3]
```diff
@@ -25,10 +27,19 @@ import './index.scss';
 const DEFAULT_VALUE = 1;
 const MarketPlace: React.FC = () => {
     const dispatch = useDispatch();
+
     const { lots, page } = useSelector(
         (state: RootState) => state.marketplaceReducer.marketplacePage
     );
 
+    const [currentLot, setCurrentLot] = useState<Lot>();
+    const [showModal, setShowModal] = useState<boolean>(false);
```
Comment (TarasPotsikaylo) [inline web/console/src/app/views/MarketPlacePage/index.tsx]: no need space beetwen constants

File: web/console/src/app/components/common/ModalMarketplace/index.tsx (hunk 2 above)
Comment (AnnaTsybel) [reply web/console/src/app/components/common/ModalMarketplace/index.tsx]: It was written by mistake, the lot will exist in all cases

**Review by TarasPotsikaylo — APPROVED** [review]

**Pull request merged**